cerbero-uninstalled

```diff
@@ -24,7 +24,7 @@
 ARGS=$@
 SCRIPTDIR="`dirname $0`"
 
-if [ "$OSTYPE" == "msys" ]; then
+if [ "$OSTYPE" = "msys" ]; then
```

Comment:
This change seems unrelated to the main topic of the patch. Maybe we can avoid adding it in this PR?

Reply:
Yes, this is totally unrelated and it should go upstream. I'm gonna put it in a separated PR that I will also push upstream.

Reply:
Other PR with only the syntax error is here: https://github.com/Igalia/cerbero/pull/21